azure-pipelines-steps.yml

```diff
@@ -0,0 +1,17 @@
+
+steps:
+
+- script: npm install
+  displayName: 'npm install'
+
+- script: npm run build
+  displayName: 'npm run build'
+
+# test with node 6
+- task: NodeTool@0
```

Comment:
I think if we do this we should matrix across different versions of node - probably 6 and 12, maybe 5 too (we currently do 5). Note that this need isn't consistent across all our OSS repos, e.g. in typed-rest-client we currently test as low as 4 I believe

Reply:
We might also want to matrix across more hosted platforms? E.g. windows-2019